config/200-clusterrole.yaml

```diff
@@ -29,6 +29,9 @@ rules:
   - apiGroups: ["tekton.dev"]
     resources: ["eventlisteners", "triggerbindings", "triggertemplates"]
     verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
+  - apiGroups: ["tekton.dev"]
+    resources: ["eventlisteners/status", "triggerbindings/status", "triggertemplates/status"]
+    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
```

Comment:
For my own education: I thought the above lines (all verbs on eventlisteners, triggerbindings and triggertemplates) would be enough - looks like we do the same thing for pipelines https://github.com/tektoncd/pipeline/blob/master/config/200-clusterrole.yaml :O do you happen to know why we need to do this?

Reply:
I haven't been able to find formal documentation on this, but since the status is a subresource, it is also "permissioned" like a resource. In this way, I think it makes sense to require separate RBAC permissions. However, it would make sense to have implicit access since the parent access has already been specified (maybe not?). I have never checked, but I assume that when grabbing the resource internally, the status would be populated where this seems buggier than behavioral.